TerminalDocs/customize-settings/startup.md

```diff
@@ -0,0 +1,148 @@
+
+**Property name:** `launchMode`
+
+**Necessity:** Optional
+
+**Accepts:** `"default"`, `"maximized"`, `"fullscreen"`, `"focus"`, `"maximizedFocus"`
+
+**Default value:** `"default"`
+
+<br />
+
+___
+
+## Launch size
+
+### Columns on first launch
+
+This is the number of character columns displayed in the window upon first load. If `launchMode` is set to `"maximized"` or `"maximizedFocus"`, this property is ignored.
+
+**Property name:** `initialCols`
+
+**Necessity:** Optional
+
+**Accepts:** Integer
+
+**Default value:** `120`
+
+### Rows on first launch
+
+This is the number of rows displayed in the window upon first load. If `launchMode` is set to `"maximized"` or `"maximizedFocus"`, this property is ignored.
+
+**Property name:** `initialRows`
+
+**Necessity:** Optional
+
+**Accepts:** Integer
+
+**Default value:** `30`
+
+<br />
+
+___
+
+## Launch position
+
+This sets the pixel position of the top left corner of the window upon first load. On a system with multiple displays, these coordinates are relative to the top left of the primary display. If an X or Y coordinate is not provided, the terminal will use the system default for that value. If `launchMode` is set to `"maximized"` or `"maximizedFocus"`, the window will be maximized on the monitor specified by those coordinates.
+
+**Property name:** `initialPosition`
+
+**Necessity:** Optional
+
+**Accepts:** Coordinates as a string in the following formats: `","`, `"#,#"`, `"#,"`, `",#"`
+
+**Default value:** `","`
+
+<br />
+
+___
+
+## Disable dynamic profiles
+
+This sets which dynamic profile generators are disabled, preventing them from adding their profiles to the list of profiles on startup. For information on dynamic profiles, visit the [Dynamic profiles page](./../dynamic-profiles.md).
+
+**Property name:** `disabledProfileSources`
+
+**Necessity:** Optional
+
+**Accepts:** `"Windows.Terminal.Wsl"`, `"Windows.Terminal.Azure"`, and/or `"Windows.Terminal.PowershellCore"` inside an array
+
+**Default value:** `[]`
+
+<br />
+
+___
+
+## Startup Actions ([Preview](https://aka.ms/terminal-preview))
+
+This sets the list of actions to execute on startup, allowing the terminal to launch with a custom set of tabs and panes by default. These actions will be applied only if no command line arguments were supplied. The list of actions is represented by a string with the same format as *commands* in the command line arguments. For more information about the *commands* format, visit the [Command line arguments page](./../command-line-arguments.md).
+
+**Property name:** `startupActions`
+
+**Necessity:** Optional
+
+**Accepts:** String representing a list of actions to run
+
+**Default value:** `""`
+
+> [!IMPORTANT]
+> This feature is only available in [Windows Terminal Preview](https://aka.ms/terminal-preview).
```

Comment:
Add a warning block saying that these settings are not in the settings UI?

Reply:
Pushing this off until the settings UI becomes default.